### PR TITLE
Split terraform modules in separate repos pt 165295661

### PR DESCRIPTION
**Pull request by valerifilipov**
re https://www.pivotaltracker.com/story/show/165295661
needs git installed in master docker image

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -8,7 +8,7 @@
 terraform/.terraform
 !Makefile
 !requirements.txt
-!test/goss/
```
Comment (dincho) [inline .dockerignore]: What's the reason for this change ?

Comment (valerifilipov) [reply .dockerignore]: `test/terraform` needs to be in docker for integration tests in https://github.com/aeternity/terraform-aws-aenode-deploy/pull/1 to run

File: terraform/environments/main.tf [hunk 2]
```diff
@@ -3,7 +3,7 @@ variable "vault_addr" {
 }
 
 module "aws_deploy-main-ap-southeast-1" {
-  source            = "../modules/cloud/aws/deploy"
+  source            = "git::https://github.com/aeternity/terraform-aws-aenode-deploy.git"
```
Comment (lucafavatella) [inline terraform/environments/main.tf]: How is versioning handled? 🤔

Comment (lucafavatella) [reply terraform/environments/main.tf]: I mean - commit in this repo infrastructure shall link to immutable reference to the terraform-aws-aenode-deploy repo. And I see no version / tag / commit hash of terraform-aws-aenode-deploy in this line.

Comment (dincho) [reply terraform/environments/main.tf]: It defaults to HEAD, @valerifilipov perhaps we should tag module v1.0.0 (as it's production tested already) and reference it here ?

Comment (dincho) [reply terraform/environments/main.tf]: Also why not prefer the short version `github.com/aeternity/terraform-aws-aenode-deploy` ?

Comment (valerifilipov) [reply terraform/environments/main.tf]: Yes, we should tag both repos, after https://github.com/aeternity/terraform-aws-aenode-deploy/pull/1 and https://github.com/aeternity/terraform-aws-api-gateway/pull/1 get merged

Comment (dincho) [reply terraform/environments/main.tf]: You don't need CI integrations to tag it

File: .dockerignore (hunk 1 above)
Comment (valerifilipov) [reply .dockerignore]: removed as actually not used

File: terraform/environments/main.tf [hunk 3]
```diff
@@ -3,7 +3,7 @@ variable "vault_addr" {
 }
 
 module "aws_deploy-main-ap-southeast-1" {
-  source            = "../modules/cloud/aws/deploy"
+  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=v1.0.0"
```
Comment (lucafavatella) [inline terraform/environments/main.tf]: I see there is https://github.com/aeternity/terraform-aws-aenode-deploy/tree/v1.0.0

Not sure about syntax though it makes sense.
